recipes-extended/ripple/files/ripple.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Firebolt reference implementation
+After=wpeframework.service
```

Comment:
Please add the below list of thunder service dependencies 
Controller
System
UserSettings
DisplaySettings
PersistentStore
HdcpProfile
TextToSpeech